lib/go-tc/tovalidate/validate.go

```diff
@@ -43,3 +45,28 @@ func ToErrors(err map[string]error) []error {
 	}
 	return vErrors
 }
+
+// ToError converts a map of strings to errors into a single error.
+//
+// Because multiple errors are collapsed, errors cannot be wrapped and therefore
+// error identity cannot be preserved.
+func ToError(err map[string]error) error {
+	if len(err) == 0 {
+		return nil
+	}
+	var b strings.Builder
+	for key, value := range err {
+		if value != nil {
+			b.WriteRune('\'')
```

Comment:
That changes the datatype. I'd have to switch it to use `WriteString` instead of `WriteRune` - which I can do; I don't think one is faster than the other. It just makes sense to me to use `WriteRune` when I'm writing only a single rune.